atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/cleaner/Scrubber.java

```diff
@@ -568,6 +565,46 @@ private void scrubCells(
         lazyWriteMetric(AtlasDbMetricNames.SCRUBBED_CELLS, allCellsToMarkScrubbed.size());
     }
 
+    private Multimap<Cell, Long> scrubBatchOfCells(
+            TransactionManager txManager,
+            long scrubTimestamp,
+            TransactionType transactionType,
+            TableReference tableRef,
+            List<Cell> cells) {
+        try {
+            return tryScrubBatchOfCells(txManager, scrubTimestamp, transactionType, tableRef, cells);
+        } catch (IllegalArgumentException ex) {
+            if (ex.getCause() instanceof TableMappingNotFoundException) {
```

Comment:
Hmm. I think this creates an unfortunate bit of coupling with the way we set up the KVS (in particular, TableRemappingKeyValueService), and in particular might not work on Cassandra - since this class is in impl-shared, it should in theory work with KVSes in general. 

I'd suggest we plumb the trace for `TableMappingNotFoundException` anywhere in the cause stack so this will work with DbKVS even if we change the wrapping order of the KVSes in some way. This still won't work for cassandra as the "not found" thing is `InvalidRequestException` in Thrift which we don't have access to from here - might be worth leaving a comment that this doesn't work with Cassandra, but we consider this acceptable because the Scrubber is generally not used with Cassandra.